src/iexit.cpp

```diff
@@ -25,40 +22,37 @@ main( int argc, char **argv ) {
         usage( argv[0] );
         exit( 0 );
     }
-    ix = myRodsArgs.optind;
 
+    rodsEnv myEnv;
     status = getRodsEnv( &myEnv );
+    if ( myRodsArgs.verbose == True ) {
+        printf( "getRodsEnv status [%d]\n", status );
+    }
 
-    envFile = getRodsEnvFileName();
+    char* envFile = getRodsEnvFileName();
     status = unlink( envFile );
     if ( myRodsArgs.verbose == True ) {
-        printf( "Deleting (if it exists) session envFile:%s\n", envFile );
-        printf( "unlink status = %d\n", status );
+        printf( "Deleting (if it exists) session envFile: %s\n", envFile );
+        printf( "unlink status [%d]\n", status );
     }
 
-    if ( ix < argc ) {
-        if ( strcmp( argv[ix], "full" ) == 0 ) {
-            if ( myRodsArgs.verbose == True ) {
-                obfRmPw( 0 );
-            }
-            else {
-                obfRmPw( 1 );
-            }
-        }
-        else {
-            printf( "option %s is unrecognized\n", argv[ix] );
-        }
+    if ( myRodsArgs.verbose == True ) {
```

Comment:
Based on `test_iexit_verbose` in irods/irods#4263, is the verbose option supposed to wait for confirmation from the user? I don't see it here, but I could be misunderstanding the intent.

Either way, 40-41 be part of the block which starts on line 34. Or is the unlink supposed to happen after the print on line 35? In the latter case, that print could be moved to the block started on line 28.

Reply:
Yes, it does wait for confirmation.   I'll look at collapsing the if/else.